api/items_post.md

```diff
@@ -80,6 +80,41 @@ understands. The meaning of each key is explained in the comments.
       // If a message with no stack trace, use "message"
       // If an iOS crash report, use "crash_report"
 
+      // Optional: "telemetry". Only applicable if you are sending [telemetry data](/docs/notifier/rollbar.js/#telemetry). 
+      "telemetry": {
+        // Required: level
+        // The severity level of the telemetry data. One of: "critical", "error", "warning", "info", "debug".
+        "level": "info",
+
+        // Required: type
+        // The type of telemetry data. One of: "log", "network", "dom", "navigation", "error", "custom".
```

Comment:
I checked against [this PR](https://github.com/rollbar/moxapi/pull/169) and the [tech spec](https://rollbar.quip.com/n2uFATVRZWF9/Telemetry-Tech-Spec) to write these docs. I'm not sure; do you know where in the code I can find this? I've been looking but haven't found a list of accepted type values.